src/Microsoft.ML.OnnxConverter/OnnxContextImpl.cs

```diff
@@ -32,9 +33,10 @@ internal sealed class OnnxContextImpl : OnnxContext
         private readonly string _producerVersion;
         private readonly long _modelVersion;
         private readonly OnnxVersion _onnxVersion;
+        private readonly int _opSetVersion;
 
         public OnnxContextImpl(IHostEnvironment env, string name, string producerName,
-            string producerVersion, long modelVersion, string domain, OnnxVersion onnxVersion)
+            string producerVersion, long modelVersion, string domain, OnnxVersion onnxVersion, int opSetVersion = 11)
```

Comment:
nit: instead of hardcoding `11` in here, I think it's better to initialize with `opSetVersion = CurrentOpsetVersion`, so that we only need to update this number in one place, and avoid posible future mistakes.

Reply:
Good Catch!